api/envoy/config/core/v3/proxy_protocol.proto

```diff
@@ -30,6 +30,11 @@ message ProxyProtocolPassThroughTLVs {
   // TLV type is defined as uint8_t in proxy protocol. See `the spec
   // <https://www.haproxy.org/download/2.1/doc/proxy-protocol.txt>`_ for details.
   repeated uint32 tlv_type = 2 [(validate.rules).repeated = {items {uint32 {lt: 256}}}];
+
+  // Whether TLVs should be base64-encoded before being passed to filter state.
+  // This is useful in cases where the TLVs aren't UTF-8 encoded but need to be passed through without lossy sanitization.
+  // Defaults to false.
+  bool encode_tlvs = 3;
```

Comment:
one question, when other filter to ready those tlvs from the metdata, how they can know this tlvs is base64 encoded or not? since other filter won't ready proxy filter's config.

Reply:
Good point. Do you think it would be a better idea to return some flag on whether the TLVs are base64-encoded? I don't know if it would be a major API problem to have filters need to implicitly understand whether TLVs are b64-encoded. 

I also just noticed #33146 and suspect that if/when that PR merges this feature might be completely unneeded.

Reply:
> Good point. Do you think it would be a better idea to write some flag on whether the TLVs are base64-encoded? I don't know if it would be a major API problem to have filters need to implicitly understand whether TLVs are b64-encoded.
> 

we probably can put another metadata to indicate the TLVs is encoded or not.

> I also just noticed #33146 and suspect that if/when that PR merges this feature might be completely unneeded.

emm...good point also, I'm little prefer to #33146 also. then we won't have the issue I mentioned.

Reply:
So given that https://github.com/envoyproxy/envoy/pull/33146 is in progress can we close this?